{{cookiecutter.project_slug}}/docker-compose.yml

```diff
@@ -1,25 +1,32 @@
-version: '2'
+version: '3'
 services:
 
   db:
     image: mdillon/postgis
-    volumes:
-      - pgdata:/var/lib/postgresql/data/pgdata
+    ports:
+      - "5434:5432"
     environment:
-      POSTGRES_PASSWORD: 'buildout'
-      POSTGRES_USER: 'buildout'
+      POSTGRES_USER: 'nens'
+      POSTGRES_PASSWORD: 'nens'
       POSTGRES_DB: '{{ cookiecutter.package_name }}'
-      PGDATA: '/var/lib/postgresql/data/pgdata'
 
   web:
     build: .
-    command: "bin/django runserver 0.0.0.0:5000"
+    user: $UID  # run `export UID` to make this work (or add to your .bashrc)
+    command: "pipenv run python manage.py runserver 0.0.0.0:5000"
     ports:
       - "5000:5000"
     volumes:
-      - .:/code
+      # do not use .:/code to circumvent a docker issue
```

Comment:
Welk issue is dat?

Reply:
"volume name is too short, names should be at least two alphanumeric characters"